### DISCUSSION TITLE
Few questions regarding numbers Blockchain

### DISCUSSION THREAD

**Discussion by Lasheron** [numbersprotocol/community-support#128] [Q&A] [answered]
1)How will  numbers protocol Blockchain help Capture App?

2)Why numbers protocol requires it's own Blockchain?

3)What changes can we expect interms of speed gas fee and security with the change of current Blockchain technology to numbers Blockchain when the mainnet is live ?

4)Will there be bridges, Dex built by the team into the capture app in the coming future?

**Comment by ethanwu155** [comment, marked as answer]
**How will Numbers Protocol Blockchain help Capture APP?**

As you know Capture APP is powered by Numbers Protocol. Photos taken with Capture APP are registered to the Numbers Network and have blockchain records that at the very least provides us with information on creator, location, timestamp and circumstance of creation. By indexing these creations with unique ids (Numbers ID) and storing them on the blockchain we are able to reliably trace and verify these assets. This is especially important given the current trustless landscape of digital media. 

Before the launch of Numbers Blockchain, we had to rely on third party blockchains such as IOTA and Avalaunch to support our archiving efforts. Launching Numbers Blockchain because we can provide users with the full breadth of what Numbers Protocol can offer (royalty distribution, asset registration, native NFT minting).

Numbers Blockchain will also be very important with NIT facilitating any transactions and the commits themselves. Most digital assets life cycles include edits to enhance the product quality. It is paramount that these changes are chronicled and made transparent with the blockchain.

NUM, the native token of the Numbers Protocol Network, also plays a role with Capture APP. With Numbers Blockchain, users should expect some quality of life improvements in the form of lower gas fees.

**Why does Numbers Protocol require its own Blockchain?**

Numbers Protocol is the first asset-centric protocol designed for the digital content industry. It supports the entire lifecycle of modern digital content — from Web3.0 registration and decentralized storage to NFT minting and royalty sharing. With so much manipulation, theft and censorship, Numbers Protocol makes the goal of bringing trust back to digital media a top priority.

The vision of Numbers Protocol is to provide the full picture of digital content. Every media file registered with Numbers Protocol has a unique Numbers ID, which also acts as the IPFS CID. With Numbers ID anyone can access physical storage supported by IPFS and Filecoin, the most popular and reputable decentralized storage solution, as well as on-chain history. So, the next question is, where should the on-chain history be kept?
There is a perception that NFTs are the digital content on-chain history solution. Those who have closely monitored the NFT situation know that this is far from the truth. [At best, NFTs are tokens that represent ownership](https://law.asia/nfts-ip-conundrum/). In order to build a more thorough solution for digital content, we need to look at the content file itself and create an indexing system that can truly represent the content. That is why we created Numbers ID and the native blockchain to support it.
Over the past two years, Numbers Protocol has leveraged ledger and blockchains such as IOTA and Avalanche C-Chain to provide on-chain proof of content history. When Numbers Blockchain fully launches, it will replace IOTA and Avalanche C-Chain to provide a more cost-efficient and higher performing asset history solution.


**What changes can we expect in terms of speed, gass fees and security with the change of current blockchain technology to numbers Blockchain when the mainnet is live?**

You should expect lower gas fees and more control with the switch to Numbers Blockchain.

**Will there be bridges, dexes built by the team into the capture app in the coming future?**

This is definitely a possibility. Right now we are focused on pushing out features that are critical to our protocol. The community will definitely be first to know. Keep an eye on your media channels for any and all Numbers Protocol news.

**Reply by Lasheron** [reply]
Thank you for your detailed answer ☺️
